src/Storageless/Http/SessionMiddleware.php

```diff
@@ -248,25 +248,21 @@ public function extractSessionContainer(Token $token = null) : SessionInterface
     private function appendToken(SessionInterface $sessionContainer, Response $response, Token $token = null) : Response
     {
         $sessionContainerChanged = $sessionContainer->hasChanged();
-        $sessionContainerEmpty   = $sessionContainer->isEmpty();
```

Comment:
Basically that leads to the question: do we want to keep this lazy initialisation thing or not?